webpack.common.js

```diff
@@ -6,12 +6,27 @@ const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const process = require('process');
+
+const makeEndpoints = () => {
+  const { HAPPA_API_ENDPOINT, HAPPA_PASSAGE_ENDPOINT } = process.env;
+
+  return {
+    apiEndpoint: HAPPA_API_ENDPOINT
```

Comment:
I think this will break when it hits this startup script right? https://github.com/giantswarm/happa/blob/master/scripts/start.sh

Reply:
Nope, the start script operates on `index.html`. The regex does not care about whats coming after the `: `.